portfolio.html

```diff
@@ -0,0 +1,606 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link rel="stylesheet" href="portfolio.css" />
+  </head>
+  <body>
+    <header class="nav-container">
+      <nav>
+        <div class="lower-bar">
+          <div class="hamburger">
+            <img
+            src="bar/Enabled.png"
+            class="allign"
+            alt="hamburger menu symbol"
+          />
+          </div>
+          <div class="close1">
+            <p class="close-menu">&times;</p>
+          </div>
+          
+        </div>
+        <div class="contacts-menu">
+          <ul>
+            <li>
+              <a href="#section11" class="navlinks">Portfolio</a>
+              <hr />
+            </li>
+            <li>
+              <a href="#section33" class="navlinks">About</a>
+              <hr />
+            </li>
+            <li>
+              <a href="#section44" class="navlinks">Contact</a>
+              <hr />
+            </li>
+          </ul>
+        </div>
+
+        <div class="desktop-left">
+          <div class="name"><a href="#section11">Haftamu</a></div>
+          <div class="contacts">
+            <ul>
```

Comment:
- [Optional] Recommended: PLease the Logo(Haftamu) should be displayed on the nav bar as in the Figma design, but yours is missing.

##### Figma Design
![Screenshot from 2022-07-20 09-39-35](https://user-images.githubusercontent.com/58771507/179937829-ce63b400-70cc-4ef7-9488-282a2ab1d008.png)


##### Your design
![Screenshot from 2022-07-20 09-36-58](https://user-images.githubusercontent.com/58771507/179937244-32c2d73e-a973-4b32-9072-a516b5729bb2.png)